### ISSUE TITLE
Luminosity contrast ratio of the text with background color is 2.6:1 which is less than required ratio 4.5:1.

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#32518]
### Describe the bug

**Test Environment:**
OS: Windows 11 24H2 (OS Build 26100.4946_
Browser: Edge (Version 139.0.3405.125)
Tool: Color contrast Analyzer tool
URL: https://vivacommonstorybooktest.z13.web.core.windows.net/?path=/docs/components-appheader--docs

**User Impact:**
Users with visual impairment who rely on contrast of the text will find difficulty if the contrast ratio of the text is less than minimum required standards. 

**Actual Result:**
Luminosity contrast ratio of the text with background color is 2.6:1 which is less than required ratio 4.5:1.

**Refer Attachment:**
Luminosity contrast ratio of the text with background color is 2.6:1 which is less than required ratio 4.5:1.png
Luminosity contrast ratio of the text with background color is 2.6:1 which is less than required ratio 4.5:1.mp4

<img width="972" height="550" alt="Image" src="https://github.com/user-attachments/assets/2595fba1-2268-43e7-bef3-14fd49443c96" />
https://github.com/user-attachments/assets/5692d324-fb51-4458-b9d6-ad9ebb01b014

**Expected Result:**
Luminosity contrast ratio of the text with background color should be greater than or equal to required ratio 4.5:1.

### Reproduction link

https://vivacommonstorybooktest.z13.web.core.windows.net/?path=/docs/components-appheader--docs

### Reproduction steps

1. Open URL: https://vivacommonstorybooktest.z13.web.core.windows.net/?path=/docs/components-appheader--docs in edge browser.
2. App Brading page will appear.
3. Press tab key to reach to the getting started section. 
4. Open color contrast analyzer.
5. Verify whether Luminosity contrast ratio of the text with background color is 2.6:1 which is less than required ratio 4.5:1 or not.

### System

```bash
npm create storybook@latest
```

**Comment by Sidnioulz** [comment]
This issue should already be fixed on the a11y branch (cc @MichaelArestad).
